engines/bladerunner/script/ai/free_slot_a.cpp

```diff
@@ -543,8 +544,14 @@ void AIScriptFreeSlotA::FledCombat() {
 void AIScriptFreeSlotA::checkIfOnBridge() {
 	float x, y, z;
 	Actor_Query_XYZ(kActorFreeSlotA, &x, &y, &z);
+	
+	bool isEasyDifficulty = _vm->_settings->getDifficulty() == kGameDifficultyEasy;
```

Comment:
sorry for nitpicking, but can you reorganize it a bit? that lines above are not used in this case  and can you use similar style of #if as the rest of the code, like:

... checifonbridge() {
#if !BLADERUNNER_ORIGINAL_BUGS
	if ( _vm->_settings->getDifficulty() == kGameDifficultyEasy) { // Make the bridge indestructable on easy mode, unless original behavior is desired
		return;
	}
#endif
	...oldcode
}


and i think you have a typo in indestructible"

Reply:
Technically this is more of a QoL enhancement then a bug fix over the original, how do you feel about changing the name of the flag to be BLADERUNNER_ORIGINAL_BEHAVIOR.

Reply:
good point, then can you use _vm->_cutContent? we have a special version which has plenty of this kind of stuff (when adding new game to scummvm it is "Blade Runner with restored content")